lib/manager/github-actions/update.ts

```diff
@@ -11,7 +11,7 @@ export function updateDependency(
     logger.debug(`github-actions.updateDependency(): ${newFrom}`);
     const lines = fileContent.split('\n');
     const lineToChange = lines[upgrade.managerData.lineNumber];
-    const imageLine = new RegExp(/^(\s+uses = "docker:\/\/)[^"]+("\s*)$/);
+    const imageLine = new RegExp(/^(.+docker:\/\/)[^"]+(")?$/);
```

Comment:
Since trailing white spaces are allowed in the regex of `extract.ts`, you should not remove `\s*`.

```diff
- ^(.+docker:\/\/)[^"]+(")?$
+ ^(.+docker:\/\/)[^"]+("?\s*)$

Reply:
Fixed.